modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -44,6 +44,7 @@
                 </div>
             </div>
             <div class="row">
+                {if 1 < count($form.centerID.options)}
```

Comment:
It is just a coding habit to cause a parse error if I missed an operator while typing. But in this case, it would have cause an error the other way around to.